pkgs/development/interpreters/php/8.1.nix

```diff
@@ -21,7 +21,10 @@ let
       })
       # Backport of PHP_LIBXML_IGNORE_DEPRECATIONS_START and PHP_LIBXML_IGNORE_DEPRECATIONS_END
       # Required for libxml2 2.13 compatibility patch.
-      ./php81-fix-libxml2-2.13-compatibility.patch
+      (fetchpatch {
+        url = "https://github.com/php/php-src/commit/e2d97314ab342d434e778cd00a2f34e4bdb07664.patch";
+        hash = "sha256-w0hyYUgbRGpvIBfWeDTSEUGpiJdyrtNjKy+Fn1vyAO0=";
+      })
       # Fix build with libxml2 2.13+. Has to be applied after libxml2 2.12 patch.
       (fetchpatch {
         url = "https://github.com/php/php-src/commit/9b4f6b09d58a4e54ee60443bf9a8b166852c03e0.patch";
```

Comment:
While at it, please replace this with https://github.com/php/php-src/commit/4fe821311cafb18ca8bdf20b9d796c48a13ba552 (the former is not on any branch)

Reply:
It is a backport to PHP-8.2 branch. See this PR https://github.com/php/php-src/pull/14816

Reply:
https://github.com/php/php-src/commit/9b4f6b09d58a4e54ee60443bf9a8b166852c03e0

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
Right..

Reply:
My apologies I missed it.